spec/views/users_show_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  before(:each) do
+    @users = [
+      @user1 = User.create(
+        name: 'Derrick',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Fullstack Web developer from Uganda',
+        posts_counter: 9
+      ),
+      @user2 = User.create(
+        name: 'Lilly',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Poland.',
+        posts_counter: 1
+      ),
+      @user3 = User.create(
+        name: 'Tom',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Mexico.',
+        posts_counter: 0
+      )
+    ]
+  end
+
```

Comment:
- [ ] Similarly, please ensure provide the following test cases for the _User show_ view:
   * I can see the user's profile picture.
   * I can see the user's username.
   * I can see the number of posts the user has written.
   * I can see the user's bio.
   * I can see the user's first 3 posts.
   * I can see a button that lets me view all of a user's posts.
   * When I click a user's post, it redirects me to that post's show page.
   * When I click to see all posts, it redirects me to the user's post's index page.